episodes/03-collaborative-git.md

```diff
@@ -0,0 +1,102 @@
+
+---
+title: Collaborative Version Control
+teaching: 180
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::::: questions
+
+- How can I use version control to collaborate with others?
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::::: objectives
+
+- Understand the basics of collaborative version control with git and Github
+- Understand the difference between centralized and distributed workflows
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+### Instructor notes
+Teaching is done as a pair of instructors. 
+Instructor A acts as the owner of the repository, instructor B as a collaborator (internal or external).
```

Comment:
Use the instructor note box, like in here: https://github.com/carpentries-incubator/deep-learning-intro/blob/7687adfb24a93e7dd2218ce01dd4884320cbf2ff/episodes/4-advanced-layer-types.Rmd#L300

This creates a box that only shows in instructor view.

Reply:
Nice